packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -193,6 +193,7 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin
     if (!canDrag) {
       _gestureRecognizers = const <Type, GestureRecognizerFactory>{};
     } else {
+      final ScrollPhysics physics = _configuration.getScrollPhysics(context);
```

Comment:
It's not quite that simple.  See `_updatePosition` around line 125.  Probably _updatePosition should cache the physics on this object so we can just use it here.